#! /bin/sh /usr/share/dpatch/dpatch-run
## 15_hppa_ldcw.dpatch by  <lamont@bld-4.mmjgroup.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: patch from Ubuntu to properly support hppa

@DPATCH@
diff -urNad qt4-x11-4.1.0~/src/corelib/arch/parisc/q_ldcw.s qt4-x11-4.1.0/src/corelib/arch/parisc/q_ldcw.s
--- qt4-x11-4.1.0~/src/corelib/arch/parisc/q_ldcw.s	2006-02-03 01:50:30.000000000 +0000
+++ qt4-x11-4.1.0/src/corelib/arch/parisc/q_ldcw.s	2006-02-03 02:00:36.000000000 +0000
@@ -1,17 +1,7 @@
-	.SPACE $PRIVATE$
-	.SUBSPA $DATA$,QUAD=1,ALIGN=8,ACCESS=31
-	.SUBSPA $BSS$,QUAD=1,ALIGN=8,ACCESS=31,ZERO,SORT=82
-	.SPACE $TEXT$
-	.SUBSPA $LIT$,QUAD=0,ALIGN=8,ACCESS=44
-	.SUBSPA $CODE$,QUAD=0,ALIGN=8,ACCESS=44,CODE_ONLY
-	.IMPORT $global$,DATA
-	.IMPORT $$dyncall,MILLICODE
-	.SPACE $TEXT$
-	.SUBSPA $CODE$
-
+	.text
 	.align 4
 	.EXPORT q_ldcw,ENTRY,PRIV_LEV=3,ARGW0=GR,RTNVAL=GR
-q_ldcw
+	.label q_ldcw
 	.PROC
 	.CALLINFO FRAME=0,CALLS,SAVE_RP
 	.ENTRY
